DocumentedExamples/SeaIce_Obs_Model_Compare.ipynb

```diff
@@ -0,0 +1,5185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.        cice_ds['time'] = cice_ds.time.to_pandas() - timedelta(days = 1)
</pre><p>better to offset by 30min, so this will work with daily or higher-frequency data (some is 3-hourly)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>